src/app/[locale]/components/Footer.tsx

```diff
@@ -42,15 +42,18 @@ const Footer = () => {
 
   return (
     <PageFooter>
-      {links.map((link) => (
-        <Link
-          href={link.href}
-          key={link.label}
-          className="flex items-center !no-underline hover:!underline"
-        >
-          <IconChevronRight className="w-6 h-6" /> {link.label}
-        </Link>
-      ))}
+      <ul>
```

Comment:
Dit zou misschien een `<LinkList>` Component kunnen zijn? (Van [Utrecht](https://nl-design-system.github.io/utrecht/storybook-react/?path=/docs/react-link-list--docs) of [Amsterdam](https://designsystem.amsterdam/?path=/docs/components-navigation-link-list--docs)).
En ik weet niet zeker of het binnen dat component nodig is, maar de icon zou binnen een `<Icon>` Component kunnen.